app/src/main/res/values-ar/strings.xml

```diff
@@ -54,6 +54,9 @@
     <string name="button_level_1">المستوى المتقدم</string>
     <string name="button_level_2">المستوى الفظيع</string>
     <string name="button_hint">تحتاج تلميحاً؟</string>
+    <string name="button_gemini_hint">التحدث إلى Gemini</string>
+    <string name="puzzle_copied_to_clipboard">تم نسخ اللغز إلى الحافظة.</string>
+    <string name="gemini_not_installed">تطبيق Gemini غير مثبت. جارٍ فتح متجر Play.</string>
```

Comment:
@aalhour What's the proper translation for this string? I think either "Play Store" needs to be spelled out both in English, or the word "Play" needs to be in Arabic as well.